components/script/script_thread.rs

```diff
@@ -2322,7 +2322,7 @@ impl ScriptThread {
     }
 
     fn handle_css_error_reporting(&self, pipeline_id: PipelineId, filename: String,
-                                  line: usize, column: usize, msg: String) {
+                                  line: u32, column: u32, msg: String) {
```

Comment:
It’s not. https://github.com/servo/rust-cssparser/pull/157 landed in cssparser in the mean time and Servo was not updated for it.